src/main/java/mate/academy/car/sharing/service/CarService.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.car.sharing.service;
+
+import java.util.List;
+import mate.academy.car.sharing.entity.Car;
+
+public interface CarService {
+    Car add(Car car);
+
+    Car getById(Long id);
+
+    List<Car> getAll();
+
+    Car update (Car car);
+
+    void delete(Long id);
+}
```

Comment:
I think it's better to create separate AbstractService interface that would look like this: 

```
public interface AbstractService<T> {
    T add(T entity);

    T getById(Long id);

    List<T> getAll();

    T update (T entity);

    void delete(Long id);
}
```

And then extend your service from this intreface: 

```suggestion
public interface CarService extends AbstractService<Car> {
}
```